sdk-base/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPlugin.kt

```diff
@@ -12,6 +12,12 @@ import com.mapbox.maps.plugin.MapPlugin
  */
 interface CameraAnimationsPlugin : MapPlugin {
 
+  /**
+   * If debug mode is enabled extra logs will be written about animation lifecycle and
+   * some other events that may be useful for debugging.
+   */
+  var debugMode: Boolean
```

Comment:
shall we generate this config option for all the plugins? cc @kiryldz @tobrun @Chaoba

Reply:
No, I've taken a look at other plugins, there's no sense in adding it everywhere. We don't have that massive logs anywhere. That's why I closed internal tickets related to similar API for all the plugins and decided to implement it in camera only due to a lot of requests.